Library/Formula/libstfl.rb

```diff
@@ -18,6 +18,10 @@ class Libstfl < Formula
   patch :DATA
 
   def install
+    if Formula["ruby"].installed?
```

Comment:
I may be about to look like an idiot but that's happened plenty before :wink:. _As far as I know_, when it comes to special dependencies which can have multiple sources any requirement needs to have logic to prepend the `$PATH`. We do this for [emacs](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/emacs_requirement.rb#L17-L20), [Python](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/python_requirement.rb#L20-L24) and [Java](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/java_requirement.rb#L18-L23) but not [Ruby](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/requirements/ruby_requirement.rb).